cmd/crates/soroban-spec-typescript/ts-tests/src/test.ts

```diff
@@ -0,0 +1,188 @@
+import test from 'ava'
```

Comment:
I know this would be obnoxious to do, but do you think there's a benefit to staying consistent with the mocha+chai approach we have in the rest of the SDK? The minimum benefit would be developer familiarity, but maybe that's not worth porting this until it becomes a problem.

Reply:
We had more recent experience with ava and typescript tests. But now that I was able to get it working for the soroban-client repo I think we could easily port it here too.

Reply:
I like AVA, for [all the reasons](https://github.com/avajs/ava#why-ava). I don't personally see much benefit to keeping consistent tooling across repos.